integrations/purchase/bridge/bridge-api-categories.js

```diff
@@ -213,29 +213,29 @@ export const BRIDGE_API_CATEGORIES = Object.entries(idToCategory).reduce(
 // To be created with the ignored=true value
 export const CATEGORIES_TO_IGNORE = [
```

Comment:
If we do this, then let us remove the lines instead of commenting them out. We can always restore them from the git history. :)